pages/sdk/protocol-kit/guides/execute-transactions.mdx

```diff
@@ -0,0 +1,286 @@
+# Execute transactions
+
+In this quickstart guide, you will create a 2 of 3 multi-sig Safe and propose and execute a transaction to send some ETH out of this Safe.
+
+To find more details and configuration options for available methods, see the [Protocol Kit reference](./protocol-kit/reference).
+
+### Prerequisites
+
+1. [Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+2. Three externally-owned accounts with Testnet ETH in at least one account
```

Comment:
Since we've hardcoded Sepolia RPC, etc in the code later on, do we want to explicitly say Sepolia ETH here?

Reply:
There is a note in the PR description where this guide will be reviewed again as it's too old. This PR is only focused on onchain identifiers to avoid too many changes with differnet contexts